examples/AdvancedConsumer/AdvancedConsumer.csproj

```diff
@@ -1,7 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Authors>Andreas Heider;Confluent Inc</Authors>
+    <Authors>Andreas Heider;Confluent Inc.</Authors>
+    <Description>Example demonstrating how to consume messages.</Description>
+    <Copyright>Copyright 2016-2017 Confluent Inc., Andreas Heider</Copyright>
+    <PackageTags>Kafka;Confluent;librdkafka</PackageTags>
```

Comment:
Unnecessary, but still informative as you say so my thought was I might as well put this info everywhere whilst i was at it with the main project. I've changed my mind though - the info is available elsewhere, comments go out of date, they aren't critical here etc. so i'll take them out.